### PR TITLE
Feature - remove create react app boilerplate

### PR DESCRIPTION
**Pull request by dustinws**

### CODE REVIEW

**Comment by Eperuza** [comment]
Looks good!

File: src/App.js [hunk 1]
```diff
@@ -1,23 +1,9 @@
...
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <h1>Hello world!</h1>
```
Comment (dustinws) [inline src/App.js:6]: Why isn't "world" capitalized? Please fix.

**Comment by nmatics** [comment]
I accept

**Review by Eperuza — APPROVED** [review]
Capitalize world and good to merge.

**Pull request merged**